src/components/GradeBookContentContainer.tsx

```diff
@@ -7,112 +7,113 @@ import GradeItemAccordion from './GradeItemAccordion';
 const ContentContainer = styled.div`
 	color: ${(props) => props.theme.colors.text[1]};
 	font-weight: bold;
-`;
+	padding: 50px 0px 50px 50px;
 
-const TitleRow = styled.div`
-	display: flex;
-	flex-direction: row;
-	flex-wrap: wrap;
-	width: 100%;
-	border-style: hidden hidden solid hidden;
-	color: ${(props) => props.theme.colors.gray[1]};
+	display: grid;
+	grid-template-columns:
+		minmax(75px, 1fr) minmax(200px, 5fr) minmax(200px, 5fr)
+		minmax(200px, 5fr) minmax(200px, 5fr);
+	grid-template-rows: 75px 75px;
```

Comment:
I needed a static min to prevent the whole thing from collapsing in on itself once you resize to smaller than a tablet. I found 75 and 200 to be the smallest possible while still maintaining proportions.